app/news/markdown.module.css

```
@@ -0,0 +1,118 @@
+.markdown ul {
+  font-size: 1.1rem;
+  font-family: "Yu Mincho";
+  color: rgb(0, 0, 0);
+  margin-bottom: 25px;
+}
+.backButton {
+  position: fixed;
+  bottom: 20px;
+  right: 10%;
+
+  width: 120px;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)